src/Repository/ProcurationProxyRepository.php

```diff
@@ -104,7 +106,6 @@ public function findMatchingProxies(ProcurationRequest $procurationRequest)
                     'pp.voteCountry = :voteCountry'
                 )
             ))
-            ->andWhere('pp.foundRequest IS NULL')
```

Comment:
do you need to replace this check by something like 
```sql
(country = 'FR' AND pp.frenchRequestAvailable)
OR (country != 'FR' AND pp.foreignRequestAvailable)
```

Reply:
yes, should be similar query that the one who count the number of match :+1: